modules/serverless-traffic-routing-config.adoc

```diff
@@ -0,0 +1,79 @@
+[id="serverless-traffic-routing-config_{context}"]
+= Configuring traffic routing
+
+When you create a Knative service, it does not have any default `traffic` spec settings. By setting the `traffic` spec, you can split traffic over any number of fixed revisions, or send traffic to the latest revision.
+
+[id="serverless-traffic-routing-config-multiple-revisions"]
+== Splitting traffic routing between multiple revisions
+
+The following example shows how the list of revisions in the `traffic` spec can be extended so that traffic is split between multiple revisions.
+
+This example sends 50% of traffic to the revision tagged as `current`, and 50% of traffic to the revision tagged as `candidate`:
+
+[source,yaml]
```

Comment:
Consider block titles because these are not code blocks in a procedure that follow instructional text from a step.